Source/PropertyTools.Wpf/DataGrid/DataGrid.Strategy.cs

```diff
@@ -201,6 +203,31 @@ public override void GenerateColumnDefinitions()
                     return;
                 }
 
+                var view = CollectionViewSource.GetDefaultView(this.ItemsSource);

```

Comment:
I think should not "this" stylebe the standart? because every itemscontrol in wpf uses this way (DataGrid, GridView, etc...)